docs/current/getting-started.md

```diff
@@ -12,305 +12,137 @@
+|  | vq server  |  |vq agent  |<----------------|          |
+|  |            |  |          |  |              |          |
+|  +------------+  +----------+  |              |          |
++--------------------------------+              +----------+
+```
 
-For working examples of both kinds of SoT, see [configurations](https://github.com/CiscoCloud/vaquero-vagrant/tree/master/config).
+**NOTE: If you get an error on the console of boot-0, "No bootable medium" it's a good indication that the VirtualBox Extension Pack was not installed (in the prerequisites above). You will need to install that, completely remove that VM (boot-0), and then run the cluster.sh command again.**
 
-**Note**: We default DHCP to run in server mode. If you want to run vaquero in DHCP proxy mode, edit the configuration in `config/` and start the dnsmasq VM by running: `vagrant up dnsmasq`. This will stand up dnsmasq VM running a DHCP server that only serves IP addresses.
+### Finish up
 
-##### git SoT:
+When you're done with the Vaquero and boot VMs that have been started with the steps above, do the following the clean up your environment:
 
-`docker run -v /vagrant/config/git-sot.yaml:/vaquero/config.yaml -v /var/vaquero/files:/var/vaquero/files -v /vagrant/provision_files/secret:/vaquero/secret --net="host" -e VAQUERO_SHARED_SECRET="<secret>" -e VAQUERO_SERVER_SECRET="<secret>" -e VAQUERO_SITE_ID="test-site" -e VAQUERO_AGENT_ID="test-agent" shippedrepos-docker-vaquero.bintray.io/vaquero/vaquero:latest standalone --config /vaquero/config.yaml`
+In the vs-1 VM:
+```
+# Kill the running Vaquero server/agent
+ctrl+c
 
-##### dir SoT:
+# Exit the VM
+ctrl+d
 
-`docker run -v /vagrant/config/dir-sot.yaml:/vaquero/config.yaml -v /var/vaquero/files:/var/vaquero/files -v /vagrant/local:/vagrant/local -v /vagrant/provision_files/secret:/vaquero/secret --net="host" -e VAQUERO_SHARED_SECRET="<secret>" -e VAQUERO_SERVER_SECRET="<secret>" -e VAQUERO_SITE_ID="test-site" -e VAQUERO_AGENT_ID="test-agent" shippedrepos-docker-vaquero.bintray.io/vaquero/vaquero:latest standalone --config /vaquero/config.yaml`
+# Destroy the VM
+vagrant destroy vs-1
+```
 
-## B. <a name="multiple-nodes">server/agent model (Multiple nodes)</a>
+To cleanup the boot-0 VM, right-click its entry in the VirtualBox manager and find the Power Off action under the Close menu; you can then right-click and remove the VM.
 
-### 1. clone the vagrant repo
+## Next steps
 
-`git clone https://github.com/CiscoCloud/vaquero-vagrant.git && cd vaquero-vagrant`
+At this point, you've successfully run Vaquero and provisioned a CentOS machine using VMs in your local environment. In "standalone mode", the Vaquero server and agent run together in a single VM (vs-1). In a production setting, it is more suitable to have these services split out. It's also possible to run them in separate VMs, which can demonstrate how Vaquero can be used to manage agents running in different locations.
```

Comment:
> It's also possible to run them in separate VMs, which can demonstrate how Vaquero can be used to manage agents running in different locations.

If we're going to jump into baremetal, I'd probably just take the quoted bit out or rephrase it in a way that's clear that baremetal is production and VMs is not.